276 email notifications from prp

kosior opened the pull request:
### This PR completes #276 

##### Feature list
* _Describe the list of features from Django_

* _Describe the list of features from Polymer_

##### Progress checker
- [ ] Django

- [ ] Polymer

- [ ] Django test cases

- [ ] Polymer test cases

n1207n commented on django_api/django_api/apps/indicator/models.py:
```diff
@@ -718,6 +722,38 @@ def disaggregation_values(self, id_only=False, filter_by_id__in=None,
+            'status': instance.get_report_status_display()
+        }
+
+        for person in pd.unicef_officers.all():
```
Since there won't be UNICEF authorized officers in cluster world, we should I guess send emails to IMOs if cluster indicator report's status is changed? @chachra?

kosior commented:
@n1207n I think I fixed those issues. Added notifications for IMOs but only for cluster objective and cluster activity. Not sure what to do with partner project and partner activity.

n1207n commented:
@kosior,

You do PartnerActivity only, since PartnerProject is not used for reporting. PartnerActivity has foreignkey to ClusterActivity. So you can look up like `content_object.cluster_activity.cluster_objective.cluster`
